unit_tests_script.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# need to run with root
+mkdir /mnt/outputs/
+mkdir /mnt/tmp/
```

Comment:
How are the tests managing to get at these *outside* the containers?

Reply:
The mounting tests require these to exist. If we want to change this, we could consult the WDL people. I'm not too sure what the replacement should be though.
https://github.com/openwdl/wdl/blob/20ff54d261e44e903d94149eb1f5c4dd141d26ea/SPEC.md?plain=1#L4319-L4333
https://github.com/openwdl/wdl/blob/20ff54d261e44e903d94149eb1f5c4dd141d26ea/SPEC.md?plain=1#L4368-L4383

Reply:
Removed, since the WDL people have already said that the mount points should not exist